src/checker.rs

```diff
@@ -82,6 +87,17 @@ impl Checker for CppCode {
         let code = &code[node.start_byte()..node.end_byte()];
         AC.is_match(code)
     }
+
+    #[inline(always)]
+    fn is_else_if(node: &Node) -> bool {
```

Comment:
The three next functions look pretty similar, probably we could use a macro which will take the node kind as argument.